server/src/app.test.ts

```diff
@@ -86,8 +104,9 @@ describe('errors', () => {
 
   test('returns a generic 500 response when an endpoints throws an error', async () => {
     // calling the /api/token endpoint without proper mocking throws TypeError
+    const expectedError = { error: 'client.createUserAndToken is not a function' };
```

Comment:
This makes it seem like we are throwing this specific error message on purpose, whereas the reason why this is the error message is because we are setting `({} as CommunicationIdentityClient)` as the identityClient if the environment is 'test'. This doesn't actually apply in a production environment. IMO we don't need to check for this specific error message.

Reply:
If we think that unnecessary to check the error message, I can simply check the status code only.

Reply:
That works, and we can also check the structure of the response:
```
expect(JSON.parse(getResponse.text)).toHaveProperty('error');
```